README.md

```diff
@@ -79,3 +79,10 @@ After installing all the dependencies, rebuild your app as described in the ["Ad
 
 The `@config-plugins/react-native-webrtc` provides props for extra customization, for instance, to define the camera and microphone permissions prompt messages.
 See the [plugin API](https://github.com/expo/config-plugins/tree/main/packages/react-native-webrtc) to learn more.
+
+## Troubleshooting
+
+> Super expression must either be null or a function
+- This issue may happen when using Expo 50 and `@daily-co/react-native-webrtc` 118.
+- `event-target-shim` version must match the same one requested by `@daily-co/react-native-webrtc`: `"event-target-shim": "6.0.2"`.
+- You can check inside `package-lock.json` to be sure you are using the right version.
```

Comment:
i feel like one more step—how to resolve the issue—could be helpful here.

something like, "If your version of `event-target-shim` doesn't match the one requested by `@daily-co/react-native-webrtc`, run the following: `...`"